en/extras.xliff

```diff
@@ -0,0 +1,411 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+  <file source-language="en" datatype="plaintext" original="file.ext">
+    <body>
+      <trans-unit id="currencies.AUD">
+        <source>A$</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.BRL">
+        <source>R$</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.CAD">
+        <source>CA$</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.CNY">
+        <source>CN¥</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.EUR">
+        <source>€</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.GBP">
+        <source>£</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.HKD">
+        <source>HK$</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.ILS">
+        <source>₪</source>
```

Comment:
Do we really support all of these currencies, e.g. New Shekels or Korean wons? I thought we supported a very limited number of currencies. If that's the case, we shouldn't expose the ones we don't need.

Reply:
Yeah this is a great call. I will check with Santiago which strings we actually need, I think we just have this "because we can"... Not so much because we need it.

Reply:
Yeah, I have a feeling we can at least start without translating them, and see if anyone complains.

Reply:
Perfect, I removed that in commit [28643b6](https://github.com/mozilla-l10n/mozilla-vpn-client-l10n/pull/435/commits/28643b634f64692d62e329db9e91a21a7330d257)